persistence-cassandra/javadsl/src/test/java/com/lightbend/lagom/javadsl/persistence/PersistentEntityRefTest.java

```diff
@@ -121,7 +122,12 @@ public void testAskTimeout() throws Throwable {
       try {
         reply.toCompletableFuture().get(20, SECONDS);
       } catch (ExecutionException e) {
-        throw e.getCause();
+        Throwable cause = e.getCause();
+        assertTrue(
+            cause
+                .getMessage()
+                .startsWith("Ask timed out on [PersistentEntityRef(10)] after [1 ms]."));
```

Comment:
```suggestion
                    "Ask timed out on [PersistentEntityRef(10)] after [1 ms]. Message of type [class com.lightbend.lagom.javadsl.persistence.TestEntity$Add]. A typical reason for `AskTimeoutException` is that the recipient actor didn't send a reply."));
```

To assert the details of the actual class are not lost and _only_ the super-type is printed.

Reply:
@ignasi35 this is done now